server/games/game.py

```diff
@@ -199,6 +200,15 @@ def teams(self):
         return frozenset({self.get_player_option(player.id, 'Team')
                           for player in self.players})
 
+    @property
+    def is_multi_team(self):
+        teams = set()
```

Comment:
Couldn't this simply be `return len(self.teams) > 2`?

Reply:
I didn't know this var existed. Well if we can use it for that, then sure.

Reply:
It's defined immediately above :)